src/main/java/bio/terra/metadata/Table.java

```diff
@@ -33,6 +34,18 @@ public Table name(String name) {
         return this;
     }
 
+    public Optional<List<Column>> getPrimaryKey() {
+        // NOTE: This should only return `Optional.EMPTY`
+        // when modeling a snapshot table. Dataset tables
+        // will always have a primary key.
```

Comment:
Not necessarily

Reply:
The intent was: dataset tables will always have some (potentially-empty) list value for the primary key. Am open to suggestions for rewording it.